### PR TITLE
[stable/fairwinds-insights] Added cloud costs update cron job

### PR DESCRIPTION
**Pull request by jdesouza**
**Why This PR?**
This PR adds a new job to collect the network usage and save into the clous cost history table.

Fixes #

**Changes**
Changes proposed in this pull request:

*
*

**Checklist:**

* [x] I have included the name of the chart in the title of this PR in square brackets i.e. `[stable/goldilocks]`.
* [x] I have updated the chart version in `Chart.yaml` following Semantic Versioning.
* [x] Any new values are backwards compatible and/or have sensible default.
* [x] Any new values have been added to the README for the Chart, or `helm-docs --sort-values-order=file` has been run for the charts that support it.

### CODE REVIEW

File: stable/fairwinds-insights/values.yaml [hunk 1]
```diff
@@ -388,6 +388,25 @@ closeTicketsCronjob:
...
+  # -- Cloud costs update enabled by default
+  enabled: true
+  # -- Resources for the cloud costs update job.
+  resources:
+    limits:
+      cpu: 500m
+      memory: 2Gi
+    requests:
+      cpu: 500m
+      memory: 1.5Gi
```
Comment (rbren) [inline stable/fairwinds-insights/values.yaml]: let's set these equal

File: stable/fairwinds-insights/values.yaml [hunk 2]
```diff
@@ -388,6 +388,25 @@ closeTicketsCronjob:
...
+  # -- CRON schedules for the cloud costs update job
+  schedules:
+    - name: cloud-costs-update
+      cron: "15 0/3 * * *"
```
Comment (rbren) [inline stable/fairwinds-insights/values.yaml]: is once/day enough?

Comment (jdesouza) [reply stable/fairwinds-insights/values.yaml]: it should be: 15 */3 * * * 😬

Comment (jdesouza) [reply stable/fairwinds-insights/values.yaml]: Fixed

File: stable/fairwinds-insights/values.yaml (hunk 1 above)
Comment (jdesouza) [reply stable/fairwinds-insights/values.yaml]: fixed

File: stable/fairwinds-insights/templates/cloud-costs-update.yaml [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+                {{- toYaml $.Values.cloudCostsUpdateCronjob.resources | nindent 16 }}
+              volumeMounts:
+              - name: secrets
+                mountPath: /var/run/secrets/github
+              securityContext:
+                readOnlyRootFilesystem: true
+                allowPrivilegeEscalation: false
+                privileged: false
+                runAsNonRoot: true
+                runAsUser: 10324
```
Comment (vitorvezani) [inline stable/fairwinds-insights/templates/cloud-costs-update.yaml]: As you expose `cloudCostsUpdateCronjob.securityContext.runAsUser` I assume you want to use the variable here

File: stable/fairwinds-insights/README.md [hunk 4]
```diff
@@ -110,6 +110,10 @@ See [insights.docs.fairwinds.com](https://insights.docs.fairwinds.com/technical-
 | closeTicketsCronjob.resources | object | `{"limits":{"cpu":"500m","memory":"2Gi"},"requests":{"cpu":"500m","memory":"1.5Gi"}}` | Resources for the close tickets job. |
 | closeTicketsCronjob.schedules | list | `[{"cron":"0/15 * * * *","name":"close-tickets"}]` | CRON schedules for the close tickets job. |
 | closeTicketsCronjob.securityContext.runAsUser | int | `10324` | The user ID to run the close tickets job under. |
+| cloudCostsUpdateCronjob.enabled | bool | `true` | Cloud costs update enabled by default |
+| cloudCostsUpdateCronjob.resources | object | `{"limits":{"cpu":"500m","memory":"2Gi"},"requests":{"cpu":"500m","memory":"1.5Gi"}}` | Resources for the cloud costs update job. |
```
Comment (vitorvezani) [inline stable/fairwinds-insights/README.md]: Update docs: actual `requests.memory` is 2Gi according to `values.yaml`

Comment (jdesouza) [reply stable/fairwinds-insights/README.md]: Fixed

File: stable/fairwinds-insights/templates/cloud-costs-update.yaml (hunk 3 above)
Comment (jdesouza) [reply stable/fairwinds-insights/templates/cloud-costs-update.yaml]: Fixed

**Comment by sudermanjr** [comment]
Can you please fill out the PR description?

**Review by vitorvezani — APPROVED** [review]

**Pull request merged**